Corona-Warn-App/src/test/java/de/rki/coronawarnapp/submission/ExposureKeyHistoryCalculationsTest.kt

```diff
@@ -16,53 +18,37 @@ class ExposureKeyHistoryCalculationsTest {
         converter = object : KeyConverter {
             override fun toExternalFormat(
                 key: TemporaryExposureKey,
-                riskValue: Int
+                riskValue: Int,
+                daysSinceOnsetOfSymptoms: Int
             ) =
                 KeyExportFormat.TemporaryExposureKey.newBuilder()
-                    .setRollingStartIntervalNumber(riskValue * 10)
+                    .setRollingStartIntervalNumber(key.rollingStartIntervalNumber)
+                    .setTransmissionRiskLevel(riskValue)
+                    .setDaysSinceOnsetOfSymptoms(daysSinceOnsetOfSymptoms)
                     .build()
         }
-        instance = ExposureKeyHistoryCalculations(TransmissionRiskVectorDeterminator(), converter)
+
+        instance = ExposureKeyHistoryCalculations(
+            TransmissionRiskVectorDeterminator(),
+            DaysSinceOnsetOfSymptomsVectorDeterminator(),
+            converter
+        )
     }
 
     @Test
     fun test_limitKeyCount() {
-        val rollingStartIntervalNumber = 0
-        createKey(rollingStartIntervalNumber)
-        Assert.assertEquals(0, instance.limitKeyCount(emptyList<String>()).size)
-        Assert.assertEquals(
-            7, instance.limitKeyCount(
-                listOf(
-                    "1",
-                    "2",
-                    "3",
-                    "4",
-                    "5",
-                    "6",
-                    "7"
-                )
-            ).size
-        )
-        Assert.assertEquals(
-            14, instance.limitKeyCount(
+        val tek1 = createKey(2012, 10, 15)
```

Comment:
Proposal: I find it tricky to wrap your head around the different dates (not only in this test case but in all of them).

How about defining a `now` as the first variable in a test case (or maybe even in the setup) and trying to derive other dates in a declarative way, e.g. `now.minus(15, DAYS)`? I think this will massively help maintenance of the test cases.

Might require a change to `createKey` to take an Instant object or something like that.